### PR TITLE
XW-2014 | Fix too much white space in global nav

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

File: reference-page/index.html [hunk 1]
```diff
@@ -2176,7 +2177,7 @@
...
...p://www.w3.org/2000/svg"><g fill-rule="evenodd"><path d="M12 2.162c3.204 0 3.584.012 4.849.07 1.366.062 2.633.336 3.608 1.311.975.975 1.249 2.242 1.311 3.608.058 1.265.07 1.645.07 4.849s-.012 3.584-.07 4.849c-.062 1.366-.336 2.633-1.311 3.608-.975.975-2.242 1.249-3.608 1.311-1.265.058-1.645.07-4.849.07s-3.584-.012-4.849-.07c-1.366-.062-2.633-.336-3.608-1.311-.975-.975-1.249-2.242-1.311-3.608-.058-1.265-.07-1.645-.07-4.849s.012-3.584.07-4.849c.062-1.366.336-2.633 1.311-3.608.975-.975 2.242-1.249 3.608-1.311 1.265-.058 1.645-.07 4.849-.07zM12 0C8.741 0 8.332.014 7.052.072c-1.95.089-3.663.567-5.038 1.942C.639 3.389.161 5.102.072 7.052.014 8.332 0 8.741 0 12c0 3.259.014 3.668.072 4.948.089 1.95.567 3.663 1.942 5.038 1.375 1.375 3.088 1.853 5.038 1.942C8.332 23.986 8.741 24 12 24c3.259 0 3.668-.014 4.948-.072 1.95-.089 3.663-.567 5.038-1.942 1.375-1.375 1.853-3.088 1.942-5.038.058-1.28.072-1.689.072-4.948 0-3.259-.014-3.668-.072-4.948-.089-1.95-.567-3.663-1.942-5.038C20.611.639 18.898.161 16.948.072 15.668.014 15.259 0 12 0z"/><path d="M12 5.838a6.162 6.162 0 1 0 0 12.324 6.162 6.162 0 0 0 0-12.324zM12 16a4 4 0 1 1 0-8 4 4 0 0 1 0 8z"/><circle cx="18.406" cy="5.594" r="1.44"/></g></svg>
```
Comment (rogatty) [inline reference-page/index.html]: What's that?

**Comment by rogatty** [comment]
LGTM

File: reference-page/index.html (hunk 1 above)
Comment (hakubo) [inline reference-page/index.html]: I just run npm run build - I'll double check

Comment (hakubo) [inline reference-page/index.html]: https://www.dropbox.com/s/1bo1goldcryiscj/Screenshot%202016-09-12%2016.32.29.png?dl=0

**Comment by Warkot** [comment]
Why do we need to add this? `content-box` is the default value—https://developer.mozilla.org/en/docs/Web/CSS/box-sizing. I compared visually reference page on `master` vs on `XW-2014` and I don't see any changes (on Chrome at least).

**Comment by Warkot** [comment]
LGTM

**Pull request merged**